lisa/tools/mkfsext.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import re
+from typing import Any, cast
+
+from lisa.executable import Tool
+from lisa.operating_system import Posix
+from lisa.util import LisaException
+
+
+class Mkfsext(Tool):
+    __EXIST_FILE_SYSTEM_PATTERN = re.compile(
+        r".*appears to contain an existing filesystem", re.MULTILINE
+    )
+
+    @property
+    def command(self) -> str:
+        return self._command
+
+    @property
+    def can_install(self) -> bool:
+        return True
+
+    def mkfs(self, disk: str, command: str = "mkfs.ext4") -> None:
+        self._command = command
+        cmd_result = self.node.execute(
+            f"echo y | {self.command} {disk}", shell=True, sudo=True
+        )
+        if self.__EXIST_FILE_SYSTEM_PATTERN.match(cmd_result.stdout):
+            cmd_result = self.node.execute(
```

Comment:
If user run this case on their disks, will it damage the data? How to prevent it?

Reply:
for nvme, the disk name is easy to be distinguished from the OS disk. I will add a check to skip this step if user uses an OS disk.